From d27280b14e5c3283cd974a52c48185ff3f2b6279 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Tue, 12 Mar 2024 12:26:00 +0800
Subject: [PATCH 01/10] FROM ARCHLINUX: fix(Frontends/VirtualBox): disable
 update checks
X-Developer-Signature: v=1; a=openpgp-sha256; l=882; i=xtexchooser@duck.com;
 h=from:subject; bh=UlxRRbsD3kpJQ6GMXd2ki6L2gRVLAiXdeiZbUJgR0x0=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDGkvVLa4ZkmlTrN19vXUzH4nmstc5GFb/POa/No/c87d6
 OVdbfOyo5SFQYyLQVZMkaXIsMGbVSedX3RZuSzMHFYmkCEMXJwCMJEZZYwMB7LaelLj573aZK69
 xuHN6ruPH3y+cy1lU0dum9bKR1LXVjD8DzJ1CX0f/EWozm4OF9+sCfbyfeHhpt7+PQfEty88tFG
 DAwA=
X-Developer-Key: i=xtexchooser@duck.com; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

---
 .../Frontends/VirtualBox/src/extradata/UIExtraDataManager.cpp   | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/VBox/Frontends/VirtualBox/src/extradata/UIExtraDataManager.cpp b/src/VBox/Frontends/VirtualBox/src/extradata/UIExtraDataManager.cpp
index cf81c1b5053b..44dd62c0097d 100644
--- a/src/VBox/Frontends/VirtualBox/src/extradata/UIExtraDataManager.cpp
+++ b/src/VBox/Frontends/VirtualBox/src/extradata/UIExtraDataManager.cpp
@@ -2412,7 +2412,7 @@ QString UIExtraDataManager::preventBetaBuildWarningForVersion()
 bool UIExtraDataManager::applicationUpdateEnabled()
 {
     /* 'True' unless 'restriction' feature allowed: */
-    return !isFeatureAllowed(GUI_PreventApplicationUpdate);
+    return false;
 }
 
 QString UIExtraDataManager::applicationUpdateData()

base-commit: 81d459942a69c79bfddb8707ac6e19ab4e95ad3c
-- 
2.46.0

